### PR TITLE
Buyout button in MyEquipmet view

### PR DESCRIPTION
**Pull request by smaciejs**

### CODE REVIEW

File: src/ralph/accounts/admin.py [hunk 1]
```diff
@@ -120,18 +120,25 @@ def user_licence(self, item):
         else:
             return []
 
+    def buyout_ticket(self, item):
+        get_params = {"inventory_number": item.barcode,
```
Comment (mkurek) [inline src/ralph/accounts/admin.py]: format it like that:
```
get_params = {
    "inventory_number": item.barcode,
    "serial_number": item.sn,
    ...
}
```

File: src/ralph/accounts/admin.py [hunk 2]
```diff
@@ -120,18 +120,25 @@ def user_licence(self, item):
...
+                      "serial_number": item.sn,
+                      "model": item.model,
+                      "comment": item.buyout_date
+                      }
+        url = "?".join(
```
Comment (mkurek) [inline src/ralph/accounts/admin.py:130]: wouldn't it be better to use `string.format` to create this url?

File: src/ralph/accounts/admin.py (hunk 1 above)
Comment (smaciejs) [reply src/ralph/accounts/admin.py]: Done

File: src/ralph/accounts/admin.py (hunk 2 above)
Comment (smaciejs) [reply src/ralph/accounts/admin.py:130]: with @ar4s we decide, that this form look better, but I can change it. Should I?

**Review by Blejwi — APPROVED** [review]

**Review by szymi- — APPROVED** [review]

**Pull request merged**